package.json

```diff
@@ -28,10 +28,10 @@
     "precommit-hook": "^3.0.0",
```

Comment:
Did we not say we'd be doing away with carets due to versioning inconsistencies?